pkgs/by-name/la/lager/package.nix

```diff
@@ -0,0 +1,42 @@
+{ callPackage, stdenv, lib,
+  fetchFromGitHub,
+  boost, cmake, cereal, gcc7, immer, ncurses, pkg-config, sass, SDL2, SDL2_ttf, zug
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lager";
+  version = "0.1.0";
+  src = fetchFromGitHub {
+    owner = "arximboldi";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-KTHrVV/186l4klwlcfDwFsKVoOVqWCUPzHnIbWuatbg=";
+  };
+  buildInputs = [
+    ncurses
+    SDL2
+    SDL2_ttf
+  ];
+  nativeBuildInputs = [
+    cmake
+    gcc7
+    sass
+    pkg-config
+  ];
+  cmakeFlags = [
+    "-Dlager_BUILD_TESTS=OFF"
+    "-Dlager_BUILD_EXAMPLES=OFF"
+  ];
+  propagatedBuildInputs = [
+    boost
+    cereal
+    immer
+    zug
+  ];
+  meta = with lib; {
+    homepage    = "https://github.com/arximboldi/lager";
+    description = "library for functional interactive c++ programs";
+    license     = licenses.lgpl3Plus;
```

Comment:
This is the Nix from the `lager` repo, but the license they have there is wrong (the library is MIT)

Reply:
cc @arximboldi

Reply:
Correct, the library is actually MIT, sorry for that mistake.